.circleci/config.yml

```diff
@@ -160,15 +157,15 @@ workflows:
           filters:
             branches:
               only:
-                - master
+                - rahti-dev
```

Comment:
this can't be merged like this obviously

Reply:
Im testing that the changes work and will revert back to master